_userguides/Managed_Hosting_User_Guide.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: page
-title: IBM Blue Box Managed Hosting User Guide
+title: IBM Bluemix on OpenStack Managed Hosting User Guide
 author: Leslie Lundquist
 featured: false
 weight: 20
 dateAdded: August 17, 2016
 tags: [managed hosting, domain name, registrar, status page]
 ---
 
-## Welcome to the IBM Blue Box Managed Hosting User Guide
+## Welcome to the IBM Bluemix on OpenStack Managed Hosting User Guide
```

Comment:
Please add (formerly Blue Box) as follows:
Welcome to the IBM Bluemix on OpenStack (formerly Blue Box) Managed Hosting User Guide

These are Blue Box legacy customers, so they will not know about the product name change. Their accounts are all being ended soon, but I don't want to create too much confusion for them.